bitnami/redis/templates/metrics-sentinel-prometheus.yaml

```
@@ -0,0 +1,33 @@
+{{- if and (.Values.sentinel.metrics.enabled) (.Values.sentinel.metrics.serviceMonitor.enabled) }}
```

Parenthesis are not required + check if sentinel is enabled:
```suggestion
{{- if and .Values.sentinel.enabled .Values.sentinel.metrics.enabled .Values.sentinel.metrics.serviceMonitor.enabled }}
```

---

Done.